src/tests/issues.targets

```diff
@@ -2720,6 +2716,12 @@
         <!-- End interpreter issues -->
     </ItemGroup>
 
+    <ItemGroup Condition="'$(RuntimeFlavor)' == 'mono' and '$(RuntimeVariant)' != 'llvmaot' and '$(RuntimeVariant)' != 'llvmfullaot' ">    
+        <ExcludeList Include = "$(XUnitTestBinBase)/JIT/HardwareIntrinsics/X86/X86Base/Pause*/**">
+            <Issue>https://github.com/dotnet/runtime/pull/61707#issuecomment-973122341</Issue>
```

Comment:
is there no actual issue for tracking this ?

Reply:
I can create one. Given the trend that we are focusing on AOT with LLVM. I doubt that we will ever have any bandwidth to support this for non-LLVM scenarios.